source/playbooks/get-started-with-playbooks.rst

```diff
@@ -0,0 +1,36 @@
+Get started with Mattermost Playbooks
+=====================================
+
+Access Mattermost playbooks from the Product Menu and select **Playbooks**.
```

Comment:
Let's decide how we want to consistently introduce our product verticals:

The Channels onboarding page reads like this: "Access Mattermost Channels by selecting Channels from the Product list [icon-image] icon."

How do you feel about adopting that approach for the other 2 onboarding pages?

Reply:
I think that's reasonable. I've updated the docs.